srcpkgs/python-Cython/template

```diff
@@ -24,13 +24,18 @@ do_build() {
 	:
 }
 
+python-Cython_package() {
+	depends+=" python-setuptools"
```

Comment:
declaring the package to be a sub-package of itself seems wrong to me. Why don't you just add it to the depends of the package itself? Sub-Packages can have their depends independent from the source package.

Reply:
because python-Cython needs to have its auto-dependencies + setuptools for python2, and python3-Cython needs to have its auto-dependencies + setuptools for python3. Wouldn't putting the depends line outside of the `python-Cython_Package` also give the dependency to the python3-Cython package?

Reply:
I was mistaken about the behavior of xbps-src. I will fix it now